### DISCUSSION TITLE
Safari and docs.github

### DISCUSSION THREAD

**Discussion by lodv** [community/community#49803] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I try to go to docs.github.com in Safari I get a blank page.
Until now I haven't found a solution on the Internet.
I hope somebody recognizes this problem and has a solution.
Thank you for your help.

Lodv

**Comment by 03timothyash** [comment]
Eso es. Solo de los problemas con tú cuenta de pagos y demás.Don't worry and just start the birdiering

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: lodv ***@***.***>
Sent: Saturday, March 11, 2023 8:41:27 AM
To: community/community ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GitHub Community] Safari and docs.github (Discussion #49803)


Select Topic Area

Question

Body

When I try to go to docs.github.com in Safari I get a blank page.
Until now I haven't found a solution on the Internet.
I hope somebody recognizes this problem and has a solution.
Thank you for your help.

Lodv

**Comment by enviGit** [comment]
### Hi lodv,

There could be several reasons why you are unable to access the website. Here are some possible explanations:
- Browser cache: The browser cache stores website data to make loading faster, but sometimes it can cause issues with loading pages. Try clearing your browser cache and reloading the page.
- Browser extensions: Browser extensions can sometimes interfere with website functionality. Try disabling any browser extensions you have installed and see if that fixes the issue.
- Network issues: There may be issues with your internet connection or your network settings that are preventing you from accessing the website. Try accessing the website from a different network or device to see if the problem persists.

_If none of these solutions work, you can also try accessing the website using a different browser or contacting GitHub support for further assistance._

**Comment by bbkx226** [comment, marked as answer]
Hi lodv, here are a few things you can try to troubleshoot the issue:

1. Clear your browser cache and cookies: Sometimes, clearing your browser's cache and cookies can help fix issues like this. To do this in Safari, go to Safari > Preferences > Privacy, then click "Manage Website Data" and select "Remove All" to clear your cache and cookies.

2. Disable any browser extensions: It's possible that a browser extension may be interfering with your ability to access docs.github.com. Try disabling any extensions you have installed and see if that helps.

3. Try accessing the site in a private browsing window: Sometimes, issues like this can be caused by conflicting settings or add-ons. Opening a private browsing window in Safari can help you determine if this is the case. To open a private browsing window in Safari, go to File > New Private Window.

4. Check your internet connection: Make sure that your internet connection is stable and working properly. If you are having issues with other websites as well, it may be a problem with your internet service provider or your network.

5. Try using a different browser: If none of the above steps work, try using a different browser (e.g. Chrome, Firefox) to see if the issue persists. If it doesn't, it may be an issue with Safari specifically.

**Reply by lodv** [reply]
Hi Pawel and Brandon,

Thank you so much for your help. The only solution that helped was installing another browser (Mozilla Firefox).
So it's an Safari issue. 
Thanks again.
Lo

**Reply by bbkx226** [reply]
A pleasure, lodv.

**Comment by lodv** [comment]
The case can be closed. It's solved.
